### ISSUE TITLE
Всем новичкам пишет "части тела вашего персонажа"

### ISSUE THREAD

**Issue by KIBORG04** [TauCetiStation/TauCetiClassic#13029]
#### Что должно было произойти
Не пишет

#### Что произошло на самом деле
![image](https://github.com/TauCetiStation/TauCetiClassic/assets/26389417/8314ae88-920c-4e5c-9208-b4186bd04297)

### Шаги воспроизведения

1. Создать новую куклу
2. Всё		

### Дополнительная информация:

Каждой кукле по дефолту дают сейвфайл версию 8
![image](https://github.com/TauCetiStation/TauCetiClassic/assets/26389417/7a270a64-79dc-417f-b66f-693eaf0a8662)

Так как она меньше текущей, то шаг за шагом она мигрирует до актуальной. Хз зачем это надо для НОВЫХ кукл, но сообщение должно писаться только при миграции существующей куклы, а не новой

**Comment by LudwigVonChesterfield** [comment]
У нас в принципе очень много применяется сейвфайл модификаций к новосозданным, якобы-не-поломанным куклам, и да, описанный баг - один из симптомов этого.

**Comment by ZER0-In** [comment]
Существует и на старых версиях и на новых. На локалке постоянно дрочит эта штука при каждом запуске.

**Comment by volas** [comment]
Ошибка где-то в другом месте, приведенный код просто в существующем сейвфайле фиксит версию, если её нет.

По всей логике, мы сначала пытаемся загрузить существующие префы, 
https://github.com/TauCetiStation/TauCetiClassic/blob/21003fe8761632aa1cd9d87842396301d67335b8/code/modules/client/preferences_savefile.dm#L525-L528

и если они отсутствуют - будут использоваться дефолтные, и при сохранении сохраняться с текущей максимальной версией:
https://github.com/TauCetiStation/TauCetiClassic/blob/21003fe8761632aa1cd9d87842396301d67335b8/code/modules/client/preferences_savefile.dm#L726-L734

**Comment by volas** [comment]
А хотя, оно почему-то в обоих случаях создаёт сейвфайл не проверяя?

**Comment by volas** [comment]
Да, логика правильная, если файл не существует - мы используем дефолтные.

https://github.com/TauCetiStation/TauCetiClassic/blame/21003fe8761632aa1cd9d87842396301d67335b8/code/modules/client/preferences_savefile.dm#L336-L340

Но это для общих префов. Проблема, что персонажи имеют свою версию, и с персонажем нам похоже не проверить на момент load_saved_character(), существовал ли он до этого или нет
